### ISSUE TITLE
aws_rds_cluser global database support fails when second region is added

### ISSUE THREAD

**Issue by shawnfeldman** [terraform-providers/terraform-provider-aws#7212]
https://www.terraform.io/docs/providers/aws/r/rds_global_cluster.html

second region requires a username and password.  

aws fails to add the second region because the second region can't have master_username or master_password specified.  

i noticed the tests here https://github.com/terraform-providers/terraform-provider-aws/issues/6846 
only test a single region

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Affected Resource(s)

* aws_XXXXX

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```

### Expected Behavior

success

### Actual Behavior

```* module.rds_uw2_global.aws_rds_cluster.global_aurora: 1 error(s) occurred:

* aws_rds_cluster.global_aurora: error creatingRDS cluster: InvalidParameterCombination: Cannot specify database name for cross region replication cluster
        status code: 400, request id: b86e8e5a-4919-47b9-8b22-735e0bd25f73```

```* module.rds_uw2_global.aws_rds_cluster.global_aurora: 1 error(s) occurred:

* aws_rds_cluster.global_aurora: provider.aws: aws_rds_cluster: : "master_password": required field is not set```
### Steps to Reproduce
 
create a aws global cluster with two regions like this 

terraform apply

### Important Factoids
 2 regions required to repro

### References

<!---
Information about referencing Github Issues: https://help.github.com/articles/basic-writing-and-formatting-syntax/#referencing-issues-and-pull-requests

Are there any other GitHub issues (open or closed) or pull requests that should be linked here? Vendor documentation? For example:
--->

* #0000

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 1.60.0 of the Terraform AWS Provider, very shortly.

**Comment by bflad** [comment]
This has been released in [version 1.60.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1600-february-22-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Comment by hussfelt** [comment]
This still applies if you are creating a cluster with two regions in the same plan - without a depends on between the two different regions.

That means; The plan will fail for the second region (not having username/password supplied) and you need to re-do the apply.

I am even not able to bring the two clusters up properly as it wants to modify them in the second run.
